src/apps/integrations/loris/api/applets.py

```diff
@@ -40,16 +39,6 @@ async def start_transmit_process(
     return HTTPResponse(status_code=status.HTTP_202_ACCEPTED)
 
 
-async def visits_list(
```

Comment:
Why did you remove this endpoint altogether? It's better to change it so it uses the stored LORIS config for the applet.

Reply:
The updated endpoint is not working, attempting to call it results in an unhandled exception

![CleanShot 2024-10-07 at 14 56 45](https://github.com/user-attachments/assets/058b0d6e-bf3f-4f7d-9ba1-57b4b7c24d3b)

Reply:
Previous error was from a misconfigured integration on my local